src/commands/deploy/args.js

```diff
@@ -69,6 +69,7 @@ export const latestHelp = () => `
     -S, --scope                    Set a custom scope
     --regions                      Set default regions to enable the deployment on
     --target                       Deploy to either 'staging' or 'production' aliases
+    --no-metrics                   Disable anonymous metrics collection
```

Comment:
Let's make this a property in the global config file called `collectMetrics`, which people need to set to `false` in order to disable it.